File: lecture02/02_anagrams/template.py
#!/usr/bin/env python3

import collections
import sys

# Functions

def is_anagram(s, t):
    # TODO: Return whether or not s and t are anagrams
    return False

# Main Execution

if __name__ == '__main__':
    for line in sys.stdin:
        first, second = line.strip().split()
        print('Anagram' if is_anagram(first, second) else 'Not Anagram')

File: lecture02/02_anagrams/template.cpp
// anagrams.cpp

#include <iostream>
#include <vector>

using namespace std;

// Constants

const size_t NLETTERS = 26;

// Functions

bool is_anagram(string &s, string &t) {
    // TODO: Return whether or not s and t are anagrams
    return false;
}

// Main execution

int main(int argc, char *argv[]) {
    string first, second;

    while (cin >> first >> second) {
    	cout << (is_anagram(first, second) ? "Anagram" : "Not Anagram") << endl;
    }

    return 0;
}
